### ISSUE TITLE
define a default background for content

### ISSUE THREAD

**Issue by manolo** [vaadin/vaadin-overlay#122]
Motivation is that It's very difficult to debug in dev-tools when content has no color, specially when testing complex elements without theming. And, the main reason to skip theming is because unit tests run significantly faster e.g. in vaadin-crud test suite in local chrome takes 24sec using themed imports vs 15 when use `src` ones. 
Backdrop has a default color in default styles, hence makes sense to define a default also for content.

**Comment by web-padawan** [comment]
Note we had the issue because of setting the content background in vaadin/vaadin-context-menu#81
And it has been removed in vaadin/vaadin-context-menu#188

**Comment by platosha** [comment]
I agree, background it is a defining feature of the overlay, not usable without it. Including it in the core styles makes sense then, just like with the backdrop.

The background should be on the overlay part, I think. Both themes will define (or override) the background on the overlay part. That way the core background will interfere less with themes, and not produce a non-overridable outcome, as it was previously in vaadin/vaadin-context-menu#81.

However, generally, non-themed components are not intended to be used by anyone. So I’m not really sure if it is useful to test this way. Fixing the performance bottlenecks in the themes and testing the real themed use cases would have much more sense, IMO.
